lib/jsdom/living/nodes/RadioNodeList-impl.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const NodeListImpl = require("./NodeList-impl").implementation;
+
+class RadioNodeListImpl extends NodeListImpl {
+  get value() {
+    this._update();
+    const element = this._list.find(e => e._localName === "input" && e.type === "radio" && e.checked);
+    if (element) {
+      return element.value ? element.value : "on";
```

Comment:
Why are you excluding elements with empty string value?

Reply:
oops... fixed!
(the wpt suite doesn't check this case 😮)